utils/config/daos_server.yml

```diff
@@ -204,7 +203,7 @@
 #
 #  targets: 20
 #
-#  # Count of offload/helper xstreams per engine.
+#  # Number of I/O service threads (and network endpoints) per engine.
```

Comment:
The description above is what targets is, not nr_xs_helpers. Should be more like number of addition offload service thread used to accelerate checksum and server-side RPC dispatch.